tests/CMakeLists.txt

```diff
@@ -0,0 +1,67 @@
+cmake_minimum_required(VERSION 3.1 FATAL_ERROR)
+#policy CMP0076 - target_sources source files are relative to file where target_sources is run
+cmake_policy (SET CMP0076 NEW)
+
+set(PROJECT_NAME YOGTest)
+
+project(${PROJECT_NAME} LANGUAGES Fortran)
+
+if(NOT CMAKE_Fortran_FLAGS)
+    set(CMAKE_Fortran_FLAGS "-g -ffree-line-length-none")
+endif()
+
+
+set(NN_Module_DIR "../NN_module/")
+set(CAM_Interface_DIR "../CAM_interface/")
+
+
+
+#Add cmake directory to the environment module variable
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+
+# Find the NetCDF installations and set the relevant variables for compilation
+# Then link to executables
+# Requires more legwork as NetCDF not provided by default
+find_package(PkgConfig)
+pkg_search_module(NETCDF_FORTRAN netcdf-fortran)
+if (NETCDF_FORTRAN_FOUND)
+    set(NETCDF_LIBRARIES "${NETCDF_FORTRAN_LDFLAGS}")
+    set(NETCDF_INCLUDES "${NETCDF_FORTRAN_INCLUDE_DIRS}")
+else()
+    set(NETCDF_F90 "YES")
+    find_package(NetCDF REQUIRED)
+endif()
+pkg_search_module(NETCDF_C netcdf)
+if (NETCDF_C_FOUND)
+    list(APPEND NETCDF_LIBRARIES "${NETCDF_C_LDFLAGS}")
+    list(APPEND NETCDF_INCLUDES "${NETCDF_C_INCLUDE_DIRS}")
+endif()
+
+# set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/../)
```

Comment:
Can this commented-out line be removed?

Reply:
Yes. Done in https://github.com/m2lines/convection-parameterization-in-CAM/pull/72/commits/d8e8caf51bd3fe7032ccd690665f9970eb4c6ebd